lib/generators/graphql/templates/graphql_controller.erb

```diff
@@ -30,8 +30,10 @@ class GraphqlController < ApplicationController
       else
         {}
       end
-    when Hash, ActionController::Parameters
+    when Hash
       ambiguous_param
+    when ActionController::Parameters
+      ambiguous_param.to_unsafe_hash
```

Comment:
How about adding a comment here to explain _why_ an `_unsafe` call is appropriate here? 

```suggestion
      ambiguous_param.to_unsafe_hash # GraphQL-Ruby will validate name and type of incoming variables.
```

That way, an unsuspecting reader won't be scared by seeing `unsafe` in the source!